src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj

```
@@ -1,16 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;netstandard2.1</TargetFrameworks>
```

Please reorder to
```suggestion
    <TargetFrameworks>netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

or consider. I do not see any good reason to keep both netstanard2.0 and netstandard2.1.
Following set should cover all currently supported .NET versions.
```suggestion
    <TargetFrameworks>net6.0;net462</TargetFrameworks>
```

---

What about net8?

---

As I understand, TFMs, you can reference net6 in net7 and net8. Probably also net9.